### PR TITLE
Hackathon reviews!

### PR DESCRIPTION
**Pull request by northeastprince**

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -17,16 +17,7 @@
...
-      end
-    end
-  end
-
-  namespace :api, defaults: {format: :json} do
-    scope "/v:api_version" do
-      resources :hackathons, only: [:index, :show]
-      scope module: :hackathon do
-        resources :subscriptions, only: :create
+        resource :bulk, only: [:update, :destroy]
```
Comment (garyhtou) [inline config/routes.rb:20]: Does this work without `controller: :bulk`? I believe it would try to find a `bulks` controller, which doesn't exist

Comment (northeastprince) [reply config/routes.rb:20]: Ah, you're right

**Review by garyhtou — CHANGES_REQUESTED** [review]
Great start! Can we add the following fields to the show page?
- Logo
- Banner
- High school led?
- Modality

**Review by garyhtou — APPROVED** [review]
Looks good! Go ahead and merge after the tests pass. I think the one that's failing is just a flaky test (try re-running it)

**Pull request merged**